index.js

```diff
@@ -0,0 +1,299 @@
+      calculator.first = 0
+    }
+    switch(symbol){
+      case '+':
+        calculator.calculate()
+        calculator.operand = '+'
+        break
+      case '-':
+        calculator.calculate()
+        calculator.operand = '-'
+        break
+      case 'X':
+        calculator.calculate()
+        calculator.operand = '*'
+        break
+      case '*':
+        calculator.calculate()
+        calculator.operand = '*'
+        break
+      case '÷':
+        calculator.calculate()
+        calculator.operand = '/'
+        break
+      case '/':
+        calculator.calculate()
+        calculator.operand = '/'
+        break
+    }
+    calculator.firstValueAdded = true
+  }
+
+  Calculator.prototype.calculate = function() {
+    var calculator = this
+    if(calculator.first !==0){
+      calculator.second = calculator.display.innerHTML
+      calculator.secondValueAdded = true
+      if(calculator.second !== 0){
+        switch(calculator.operand){
+          case '+':
+            calculator.result = parseFloat(calculator.first) + parseFloat(calculator.second)
+            break
+          case '-':
+            calculator.result = parseFloat(calculator.first) - parseFloat(calculator.second)
+            break
+          case '*':
+            calculator.result = parseFloat(calculator.first) * parseFloat(calculator.second)
+            break
+          case '/':
+            calculator.result = parseFloat(calculator.first) / parseFloat(calculator.second)
+            break
+        }
+        calculator.display.innerHTML = calculator.result
+        calculator.changeFontSize()
+      }
+    }
+  }
+
+  Calculator.prototype.allClear = function() {
+    var calculator = this
+    calculator.display.innerHTML = 0
+    calculator.display.style.fontSize = "70px"
+    calculator.first = 0
+    calculator.second = 0
+    calculator.operand = ''
+    calculator.result = ''
+    calculator.firstValueAdded = false
+    calculator.secondValueAdded = false
+  }
+
+  Calculator.prototype.flipSign = function() {
+    var calculator = this
+    var toFlip = calculator.display.innerHTML
+    if(toFlip > 0){
+      calculator.display.innerHTML = '-' + toFlip
+    }else{
+      var flip = calculator.display.innerHTML.indexOf('-')
+      calculator.display.innerHTML = calculator.display.innerHTML.slice(flip+1)
+    }
+  }
+
+  Calculator.prototype.percent = function() {
+    var calculator = this
+    var perc = calculator.display.innerHTML
+    var removeIndex = perc.indexOf('%')
+    var toBeCalculated = perc.replace(/\% ?/g, "")
+    calculator.display.innerHTML = (toBeCalculated / 100).toFixed(2)
+  }
+
+  Calculator.prototype.isOperand = function(sym){
+    return sym == '+' || sym == '-' || sym == '/' || sym == '*'
+  }
+  Calculator.prototype.assignClass = function(eventKey){
+    var calculator = this.current
+    switch(eventKey){
+      case '0':
+        calculator.querySelector('#zero').classList
+        .add('calculator-button-active')
```

Comment:
everytime you bring a `.something` down to the next line its customary to indent it

```js
calculator.querySelector('#one').classList
  .add('calculator-button-active')
setTimeout(function(){
  calculator.querySelector('#one').classList
    .remove('calculator-button-active')
}, 100)
```